Makefile

```diff
@@ -9,29 +9,40 @@ setup: # Set up project
 	make python-virtualenv PYTHON_VENV_NAME=$(PROJECT_ID)
 	pip install -r application/requirements.txt
 
-build: project-config # Build project
-	make docker-build NAME=NAME_TEMPLATE_TO_REPLACE
+build: # Build image
+	make docker-build-lambda AWS_ECR=$(AWS_LAMBDA_ECR)
 
-start: project-start # Start project
+start: project-start
 
-stop: project-stop # Stop project
+restart:
+	make \
+		stop \
+		start
 
-restart: stop start # Restart project
+stop: project-stop
 
-log: project-log # Show project logs
+log: project-log
 
-test: # Test project
+unit-test: # Test project
```

Comment:
We've fixed make run-unit-test now so this is not necessary

Reply:
Updated to reflect